hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -205,10 +210,10 @@ private Blockchain stubbedBlockchain() {
 	private ProcessableBlockHeader stubbedBlockHeader(long timestamp) {
 		return new ProcessableBlockHeader(
 				Hash.EMPTY,
-				Address.ZERO, //Coinbase might be the 0.98 address?
+				Address.fromHexString(asSolidityAddressHex(txnCtx.submittingNodeAccount())),
 				Difficulty.ONE,
 				0,
-				12_500_000L,
+				properties.maxGas(),
```

Comment:
Same comment here. You have to modify `long gasLimit = 1_000_000L; /` on line 139 as-well